v3/src/components/graph/plots/dot-plot/dot-plot-model.ts

```diff
@@ -29,3 +32,43 @@ export const DotPlotModel = PlotModel
       return true
     }
   }))
+  .views(self => ({
+    countAdornmentValues({ cellKey, movableValues, primaryAxisDomain }: ICountAdornmentValuesProps):
+        ICountAdornmentValues {
+      if (!movableValues || movableValues.length === 0 || !primaryAxisDomain) {
+        return self._countAdornmentValues({ cellKey })
+      }
+      movableValues = movableValues.filter(v => inRange(v, primaryAxisDomain[0], primaryAxisDomain[1]))
+      const dataConfig = self.dataConfiguration,
+        dataset = dataConfig?.dataset,
+        showMeasuresForSelection = !!dataConfig?.showMeasuresForSelection,
+        subPlotCaseIds = dataConfig?.subPlotCases(cellKey) ?? [],
+        totNumInSubPlot = subPlotCaseIds.length,
+        numSelInSubPlot = dataConfig?.filterCasesForDisplay(dataConfig?.subPlotCases(cellKey)).length ?? 0,
+        primaryAttrId = dataConfig?.attributeID(dataConfig?.primaryRole ?? "x"),
+        numericValuesForSubPlot = ((primaryAttrId && dataset)
+          ? subPlotCaseIds.filter(id => !showMeasuresForSelection || dataConfig?.selection.includes(id))
+            .map(id => dataDisplayGetNumericValue(dataset, id, primaryAttrId, false))
+          : []).filter(value => isFiniteNumber(value)),
+        [ min, max ] = primaryAxisDomain
+
+      // We iterate through the movableValues computing the count to the left and the fraction position on the axis
+      const theValues: INumDenom[] = []
+      for (let i = 0; i <= movableValues.length; i++) {
+        const prevValue = i === 0 ? -Infinity : movableValues[i - 1],
+          thisValue = i < movableValues.length ? movableValues[i] : Infinity,
+          startFraction = i === 0 ? 0 : (prevValue - min) / (max - min),
+          endFraction = i === movableValues.length ? 1 : (thisValue - min) / (max - min),
+          numerator = numericValuesForSubPlot.filter(v => v > prevValue && v <= thisValue).length
```

Comment:
Is this how v2 performs the same computation? I would have thought that:
```suggestion
          numerator = numericValuesForSubPlot.filter(v => v >= prevValue && v < thisValue).length
```
would be more conventional.